src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+
+  &__wrapper {
+    padding: 50px 40px;
+    display: flex;
+    flex-wrap: wrap;
+    width: 944px;
```

Comment:
the checklist, p.13

Reply:
I implemented the function, but I don’t understand why it is needed if our task says 4 blocks wide. It would be more correct to simply set the max width, or automatically find out the number of cards, and not manually, like mine.